template/services/discord_api_wrapper.py

```diff
@@ -0,0 +1,58 @@
+import os
+import aiohttp
+import bittensor as bt
+from typing import Optional
+
+
+BASE_URL = "daturadiscordapi.us-east-1.elasticbeanstalk.com"
+
+
+class DiscordAPIClient:
```

Comment:
Get channels need from direct api. not from miners

Reply:
We have that in API side, I will remote it from here.